tests/e2e/specs/url.cy.js

```diff
@@ -35,4 +35,21 @@ describe('URL handling', () => {
         expect(url.endsWith('?a=1&b=2')).to.be.true
       })
   })
+
+  it('reroutes to noAuth page if user isnt authenticated', () => {
```

Comment:
Difference between authorisation and authentication is basically
- Authentication is ensuring the identity of the user
- Authorisation is determining the permissions of the user

In this case the user is authenticated but not authorised to view anything
```suggestion
  it('reroutes to noAuth page if user isnt authorised', () => {
```